core/src/wallet/cosmos/CosmosLikeAccount.hpp

```diff
@@ -150,6 +154,7 @@ namespace ledger {
                         private:
                                 std::shared_ptr<CosmosLikeAccount> getSelf();
 
+                                std::shared_ptr<cosmos::Account> _accData;
                                 std::shared_ptr<CosmosLikeKeychain> _keychain;
                                 std::string _accountAddress;
```

Comment:
This can probably be removed and `_accData.address` used instead?

Reply:
I'll check. I'm worried that the address is necessary for initial synchronization, and accData is not valid until after first sync :thinking: